content/docs/tasks/traffic-management/custom-gateway/index.md

```diff
@@ -0,0 +1,355 @@
+---
+title: Add a new Custom IngressGateway
+description: Describes how to add a new custom ingress gateway manually.
+weight: 89
+keywords: [ingress,traffic-management]
+---
+
+This post provide instructions to create manually a new custom ingress gateway with automatic provisioning of certificate based on cert-manager.
+
+This post was validated on AWS Infrastructure
+
+## Prerequisites
+
+Have cert-manager installed with [chart](https://github.com/helm/charts/tree/master/stable/cert-manager)
+
+## Check cert-manager installation
+
+Cert-manager needs to be installed, if it's has been install with helm you can check installation has bellow:
+
+{{< text bash >}}
+$ helm ls
+NAME         	REVISION	UPDATED                 	STATUS  	CHART                    	APP VERSION 	NAMESPACE
+istio        	1       	Thu Oct 11 13:34:24 2018	DEPLOYED	istio-1.0.2              	1.0.2       	istio-system
+cert        	1       	Wed Oct 24 14:08:36 2018	DEPLOYED	cert-manager-v0.6.0-dev.2	v0.6.0-dev.2	istio-system
+{{< /text >}}
+
+## Create cluster issuer
+
+{{< warning_icon >}} Adapt cluster issuer provider with your own configuration, in our example we use `route53`
```

Comment:
Indent text with 4 spaces to avoid breaking the numbered list. To fix grammar, I suggest:
```suggestion
{{< warning_icon >}} Change the cluster's issuer provider with your own configuration values. The example uses the values under `route53`.
```